service/api/service_impl.go

```diff
@@ -124,7 +124,7 @@ func (s serviceImpl) notifyRemoteImmediateTaskAsync(_ context.Context, req xdbap
 			},
 		})
 
-		request := apiClient.DefaultAPI.InternalApiV1XdbNotifyImmediateTasksPost(ctx)
+		request := apiClient.DefaultApi.InternalApiV1XdbNotifyImmediateTasksPost(ctx)
```

Comment:
looks like you downgraded the openAPI version: https://github.com/xdblab/xdb-apis/pull/11/files#diff-c597776acfcaa3e2e390ec8ba7cc57f1bf36da340c29c43380318f5836e073df

Lets put it back. You need to upgrade your local version of generator